bodo/io/_fs_io.cpp

```diff
@@ -197,46 +191,6 @@ get_reader_file_system(std::string file_path, std::string s3_bucket_region,
             CHECK_ARROW_AND_ASSIGN(result, "GcsFileSystem::Make", fs,
                                    std::string("GCS"));
         }
-    } else if (is_hf) {
```

Comment:
`get_reader_file_system` is only used for Puffin files now, which doesn't need HuggingFace obviously.